addons/website_sale/static/src/js/notification/add_to_cart_notification/add_to_cart_notification.js

```diff
@@ -15,7 +15,7 @@ export class AddToCartNotification extends Component {
                     quantity: Number,
```

Comment:
should we change it?
```suggestion
                    added_qty: Number,
```